app/app-message-service.coffee

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+class MessageObject
```

Comment:
yep I mean having the class in another file and then creating instances on demand